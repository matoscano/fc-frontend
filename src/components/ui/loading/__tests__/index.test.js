import React from "react";
import { render } from "@testing-library/react";
import Loading from "../index";

it("renders a Loading", () => {
  const { container } = render(<Loading />);
  expect(container).toMatchSnapshot();
});
