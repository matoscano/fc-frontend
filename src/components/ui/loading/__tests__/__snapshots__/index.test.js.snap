// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Loading 1`] = `
<div>
  <section
    class="sc-AxjAm jhiTqR"
  >
    <div
      class="sc-AxirZ febGTa"
    >
      <div
        class="sc-AxhCb cPsxtN"
      />
      <div
        class="sc-AxiKw kdkRpp"
      >
        Loading...
      </div>
    </div>
  </section>
</div>
`;
